setup.py

```diff
@@ -27,6 +27,11 @@
     package_version = re.compile(
         r".*__version__ = '(.*?)'", re.S).match(v_file.read()).group(1)
 
+
+requirements = [
+    'psutil>=4.0.0',  # used to find processes leaked during termination
```

Comment:
It's not our bug, but it looks unusual that it doesn't fail to compile for PyPy3.